server/jetstream_api.go

```diff
@@ -581,6 +589,32 @@ func (s *Server) getRequestInfo(c *client, raw []byte) (pci *ClientInfo, acc *Ac
 	return &ci, acc, hdr, msg, nil
 }
 
+func (a *Account) trackAPI() {
+	a.mu.RLock()
+	jsa := a.js
+	a.mu.RUnlock()
+	if jsa != nil {
+		jsa.mu.Lock()
+		jsa.usage.api++
+		jsa.apiOk++
+		jsa.sendClusterUsageUpdate()
+		jsa.mu.Unlock()
+	}
+}
+
+func (a *Account) trackAPIErr() {
+	a.mu.RLock()
+	jsa := a.js
+	a.mu.RUnlock()
+	if jsa != nil {
+		jsa.mu.Lock()
+		jsa.usage.err++
```

Comment:
One could argue that this is still an API request, so we could have jsa.usage.api++ too (assuming that usage.api does not mean usage.apiOk, but simply the number of api calls).

Reply:
That is the way I had it. Will let @ripienaar weigh in when he wakes up.

Reply:
yeah I think increment both here

Reply:
So then should not be called Ok. Right now in user facing results we have..

```go
API : {
  Ok: 22,
  Err: 1
}
```

We should probably name it something else yes?

Reply:
LMK your thoughts and I can update.

Reply:
yeah good call, so make it Count or similar

Reply:
If user facing of `usage.api` has the meaning of API OK, then no, I would not bump it. I was thinking that this was just the number of API calls. I was confused because I saw this for the "api OK" case:
```
jsa.usage.api++
jsa.apiOk++
```
so I thought they were different counters.

Reply:
How about this..

```go
API : {
  Total: 22,
  Errors: 1
}
```

Reply:
Sure.

Reply:
ok will make that change.

Reply:
ok updated, please take a look when you have a moment.

Reply:
LGTM